env.d/development/common.dist

```diff
@@ -45,3 +45,6 @@ OIDC_AUTH_REQUEST_EXTRA_PARAMS={"acr_values": "eidas1"}
 AI_BASE_URL=https://openaiendpoint.com
 AI_API_KEY=password
 AI_MODEL=llama
+
+# Collaboration
+COLLAB_SERVER_URL=ws://localhost:4444
```

Comment:
I wouldn't shorten it:
COLLABORATION_SERVER_URL

Reply:
Done: https://github.com/numerique-gouv/impress/pull/425/commits/f10740acdc673888a3fa18f1572e95ded75200b4